src/esp/physics/bullet/BulletRigidObject.cpp

```diff
@@ -382,14 +334,19 @@ void BulletRigidObject::constructRigidBody(bool kinematic) {
   }
 
   //! Bullet rigid body setup
+  auto motionState = (mt == MotionType::STATIC) ? nullptr : &(btMotionState());
+
   btRigidBody::btRigidBodyConstructionInfo info =
-      btRigidBody::btRigidBodyConstructionInfo(mass, &(btMotionState()),
+      btRigidBody::btRigidBodyConstructionInfo(mass, motionState,
                                                bObjectShape_.get(), bInertia);
 
   info.m_friction = tmpAttr->getFrictionCoefficient();
   info.m_restitution = tmpAttr->getRestitutionCoefficient();
   info.m_linearDamping = tmpAttr->getLinearDamping();
   info.m_angularDamping = tmpAttr->getAngularDamping();
+  if (bObjectRigidBody_ != nullptr) {
+    info.m_startWorldTransform = bObjectRigidBody_->getWorldTransform();
```

Comment:
btTransform doesn't initialize to identify or anything else by default. It seems dangerous to not initialize it for the (bObjectRigidBody_ == nullptr) case.

Reply:
`info.m_startWorldTransform` is initialized to identity in the constructor: https://pybullet.org/Bullet/BulletFull/btRigidBody_8h_source.html#l00119.
Providing a `btMotionState` will override this transform also. Really this could be reduced to the STATIC case only, but this condition seemed more general.